{
  "uid" : "eed5ac6bfad40968",
  "name" : "test_easy_sum_set[array_a1-array_c1-array_b1]",
  "fullName" : "src.test.java.algorithms.EasySumSetProblem.test_easy_sum_set#test_easy_sum_set",
  "historyId" : "55bda4dbd0f6667ef50b08805eec898c",
  "time" : {
    "start" : 1618612338962,
    "stop" : 1618612338962,
    "duration" : 0
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: assert 2 == 1\n +  where 2 = len(['2', '3'])\n +  and   1 = len({'3'})\n +    where {'3'} = calculate_b([3], [5, 6])",
  "statusTrace" : "array_a = [3], array_c = [5, 6], array_b = ['2', '3']\n\n    @pytest.mark.parametrize(\"array_a,array_c,array_b\", test_data)\n    def test_easy_sum_set(array_a: List[int],\n                          array_c: List[int], array_b: List[str]):\n        print(f\"expected = {array_b}\")\n        print(f\"actual = {calculate_b(array_a, array_c)}\")\n>       assert len(array_b) == len(calculate_b(array_a, array_c))\nE       AssertionError: assert 2 == 1\nE        +  where 2 = len(['2', '3'])\nE        +  and   1 = len({'3'})\nE        +    where {'3'} = calculate_b([3], [5, 6])\n\nsrc/test/java/algorithms/EasySumSetProblem/test_easy_sum_set.py:20: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: assert 2 == 1\n +  where 2 = len(['2', '3'])\n +  and   1 = len({'3'})\n +    where {'3'} = calculate_b([3], [5, 6])",
    "statusTrace" : "array_a = [3], array_c = [5, 6], array_b = ['2', '3']\n\n    @pytest.mark.parametrize(\"array_a,array_c,array_b\", test_data)\n    def test_easy_sum_set(array_a: List[int],\n                          array_c: List[int], array_b: List[str]):\n        print(f\"expected = {array_b}\")\n        print(f\"actual = {calculate_b(array_a, array_c)}\")\n>       assert len(array_b) == len(calculate_b(array_a, array_c))\nE       AssertionError: assert 2 == 1\nE        +  where 2 = len(['2', '3'])\nE        +  and   1 = len({'3'})\nE        +    where {'3'} = calculate_b([3], [5, 6])\n\nsrc/test/java/algorithms/EasySumSetProblem/test_easy_sum_set.py:20: AssertionError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "a34da286ce599b48",
      "name" : "stdout",
      "source" : "a34da286ce599b48.txt",
      "type" : "text/plain",
      "size" : 37
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "src.test.java.algorithms.EasySumSetProblem"
  }, {
    "name" : "suite",
    "value" : "test_easy_sum_set"
  }, {
    "name" : "host",
    "value" : "fv-az193-651"
  }, {
    "name" : "thread",
    "value" : "3142-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "src.test.java.algorithms.EasySumSetProblem.test_easy_sum_set"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "array_a",
    "value" : "[3]"
  }, {
    "name" : "array_b",
    "value" : "['2', '3']"
  }, {
    "name" : "array_c",
    "value" : "[5, 6]"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "eed5ac6bfad40968.json",
  "parameterValues" : [ "[3]", "['2', '3']", "[5, 6]" ]
}